gpiozero/output_devices.py

```diff
@@ -1185,24 +1186,37 @@ class Motor(SourceMixin, CompositeDevice):
         :class:`DigitalOutputDevice` instances, allowing only direction
         control.
 
+    :param bool enable_is_pwm:
+        If :data:`True`, construct a :class:`PWMOutputDevice` instance for
+        the enable pin, allowing variable speed control using the enable pin
+        and not the motor controller pins. Additionally, if this is
```

Comment:
Perhaps "not the motor controller pins" should be "rather than the forward and backward pins" ? (since enable is also one of the motor controller pins? :thinking: )